### PR TITLE
Adding choice speak_to_neighbor

### PR DESCRIPTION
**Pull request by rentphoto**
Please review this, @casey-collab

### CODE REVIEW

File: russian/speak_to_neighbor/speak_to_neighbor.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Я вышел на лестничную площадку и позвонил соседям.
```
Comment (casey-collab) [inline russian/speak_to_neighbor/speak_to_neighbor.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by rentphoto** [comment]
Please review this, @casey-collab

File: russian/zefir.md [hunk 2]
```diff
@@ -53,3 +53,5 @@
 [Почесать в подмышечной впадине.](scratch/scratch.md)
 
 [Пойду в другую комнату](next_room/next_room.md)
+
+[Поговорю с соседями](speak_to_neighbor)
```
Comment (casey-collab) [inline russian/zefir.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: russian/speak_to_neighbor/speak_to_neighbor.md (hunk 1 above)
Comment (casey-collab) [inline russian/speak_to_neighbor/speak_to_neighbor.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by rentphoto** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :smile:

**Pull request merged**